settings.py

```diff
@@ -183,7 +183,26 @@
 # ex ADPOD_SLOTS = [1,2,3], will create 1st, 2nd and 3rd slot of adpod
 # ex ADPOD_SLOTS = [4,5], will create 4th and 5th slot of adpod
 # Slot numbers should be in incremental order.
-ADPOD_SLOTS = [1,2,3]
+ADPOD_SLOTS = [1]
+
+# ENABLE_DEAL_LINEITEM
+# THis option is only for Adpod setup
+# Set ENABLE_DEAL_LINEITEM = True for creating deal line item
+# Defaults to False
+ENABLE_DEAL_LINEITEM = False
+
+# DEAL_CONFIG - configuration for creating deal lineitem
+# THis option is only for Adpod setup
+# Provide price, dealtier prefix and mindealtier values
+# Number of LineItem created = len(prefix)*len(mindealtier) for each bidder
+# LineItem wil be created with pwt_dt(dealtier)targeting with pwt_dt value as prefix[index] + mindealtier[index] for each bidder
```

Comment:
Add a comment for `DFP_LINEITEM_TYPE` to be set as `SPONSERSHIP` for Deal lineitem setup.

Reply:
Done